src/react-distortable-imageoverlay-maplayer.js

```diff
@@ -1,7 +1,7 @@
 import { MapLayer, withLeaflet } from 'react-leaflet';
 import PropTypes from 'prop-types';
 
-import './lib/leaflet-distortableimage';
+import './node_modules/leaflet-distortable-image/dist/Leaflet.DistortableImage.js';
 import './lib/leaflet-path-transform';
 import L from 'leaflet';
 
```

Comment:
So in `Props` below, I think we would need to follow guidance in https://github.com/publiclab/Leaflet.DistortableImage/pull/132/ and add `suppressToolbar: true`, but I'm not 100% sure how this gets fed into the multiple instances below of `new L.DistortableImageOverlay(props.url, this.getOptions(props));` - i see three in this file, for example:

https://github.com/ChrisLowe-Takor/react-leaflet-distortable-imageoverlay/pull/4/files#diff-4d50686480707aba7ccb6620f6e54d9aR19